src/backend/catalog/pipeline_query.c

```diff
@@ -548,10 +550,11 @@ GetContQueryForId(Oid id)
 	cq->type = row->type == PIPELINE_QUERY_TRANSFORM ? CONT_TRANSFORM : CONT_VIEW;
 
 	cq->relid = row->relid;
-	cq->relname = get_rel_name(row->relid);
+	cq->name = makeRangeVar(get_namespace_name(get_rel_namespace(row->relid)), get_rel_name(row->relid), -1);
 	cq->seqrelid = row->seqrelid;
 	cq->matrelid = row->matrelid;
 	cq->active = row->active;
+	cq->osrel = makeRangeVar(get_namespace_name(get_rel_namespace(row->osrelid)), get_rel_name(row->osrelid), -1);
```

Comment:
Better to use `relid` here and use `heap_open_extended` downstream. `relid` can never change even if the relation is renamed etc.